scripts/linux/cleanup-k8s-and-cni.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
This script seems unrelated to this PR. Can you please remove it and open another one? Thanks!

Don't worry about adding a new commit, I'll squash everything before merging :)

Reply:
done